autogen/main/variables.tf.tmpl

```diff
@@ -103,6 +103,17 @@ variable "service_external_ips" {
 }
 
 {% if autopilot_cluster != true %}
+variable "insecure_kubelet_readonly_port_enabled" {
+  type = string
```

Comment:
I might have already asked, but can we use a `bool` with a default of `null` for this?

Reply:
Yes, I think that's a good idea (for this module). Will make that update.